src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -174,12 +174,7 @@ public enum Key {
          *
          */
         SearchRespectPermissionRoot,
-        /**
-         * Solr hostname and port, such as "localhost:8983".
-         * @deprecated New installations should not use this database setting, but use {@link JvmSettings#SOLR_HOST}
-         *             and {@link JvmSettings#SOLR_PORT}.
-         */
-        @Deprecated(forRemoval = true, since = "2022-12-23")
```

Comment:
Why was this deprecation notice removed? I assume it's there for a reason.

Reply:
merge error